.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
```

Comment:
Hi @ccnlui 
thanks for looking into this. We need to show that the library can still build on JDK8 (i.e. not requiring any newer JDK APIs). 
Instead of changing the existing pipeline, please add another parallel job, so that the library builds on both JDK8 and JDK17.